apps/react-18-tests-v9/src/index.tsx

```diff
@@ -1,10 +1 @@
-import * as React from 'react';
-import { createRoot } from 'react-dom/client';
-import { App } from './App';
-
-const root = createRoot(document.getElementById('root') as HTMLElement);
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
+export {};
```

Comment:
@marcosmoura by removing completely this content makes `start` command to render empty app, is this expected?